### PR TITLE
BAYC example added

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

File: Ethereum/ethereum-bayc/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# To improve query performance, we strongly suggest adding indexes to any field that you plan to filter or sort by
+# Add the `@index` or `@index(unique: true)` annotation after any non-key field
+# https://academy.subquery.network/build/graphql.html#indexing-by-non-primary-key-field
+
+type Transfer @entity {
+  id: ID!
+  from: String!
+  to: String!
+  tokenId: BigInt!
```
Comment (jamesbayly) [inline Ethereum/ethereum-bayc/schema.graphql:9]: Should this map to a BoredApe?

File: Ethereum/ethereum-bayc/schema.graphql [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# To improve query performance, we strongly suggest adding indexes to any field that you plan to filter or sort by
+# Add the `@index` or `@index(unique: true)` annotation after any non-key field
+# https://academy.subquery.network/build/graphql.html#indexing-by-non-primary-key-field
+
+type Transfer @entity {
```
Comment (jamesbayly) [inline Ethereum/ethereum-bayc/schema.graphql:5]: Can you add the date?

File: Ethereum/ethereum-bayc/schema.graphql [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  to: String!
+  tokenId: BigInt!
+  blockNumber: BigInt!
+  transactionHash: String!
+}
+
+type BoredApe @entity {
+  id: ID!
+  creator: String!
+  newOwner: String!
```
Comment (jamesbayly) [inline Ethereum/ethereum-bayc/schema.graphql:17]:
```suggestion
  currentOwner: String!
```

File: Ethereum/ethereum-bayc/schema.graphql [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  tokenURI: String!
+  blockNumber: BigInt!
+}
+
+type Property @entity {
```
Comment (jamesbayly) [inline Ethereum/ethereum-bayc/schema.graphql:22]: What's the relationship between Property and BoredApe?

File: Ethereum/ethereum-bayc/project.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    options:
+      abi: bayc
+      address: '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D'
+    assets:
+      bayc:
+        file: ./abis/bayc.abi.json
+    mapping:
+      file: ./dist/index.js
+      handlers:
+        - kind: ethereum/LogHandler
```
Comment (jamesbayly) [inline Ethereum/ethereum-bayc/project.yaml:34]: Can you add a Mint handler?

**Review by jamesbayly — CHANGES_REQUESTED** [review]

File: Ethereum/ethereum-bayc/project.yaml (hunk 5 above)
Comment (bgodlin) [reply Ethereum/ethereum-bayc/project.yaml]: https://github.com/subquery/ethereum-subql-starter/pull/42/commits/787557af34849d9c5b0b001f6c3ff4fbb68c1319

File: Ethereum/ethereum-bayc/schema.graphql (hunk 4 above)
Comment (bgodlin) [reply Ethereum/ethereum-bayc/schema.graphql]: https://github.com/subquery/ethereum-subql-starter/pull/42/commits/787557af34849d9c5b0b001f6c3ff4fbb68c1319

File: Ethereum/ethereum-bayc/schema.graphql (hunk 3 above)
Comment (bgodlin) [reply Ethereum/ethereum-bayc/schema.graphql]: https://github.com/subquery/ethereum-subql-starter/pull/42/commits/787557af34849d9c5b0b001f6c3ff4fbb68c1319

File: Ethereum/ethereum-bayc/schema.graphql (hunk 2 above)
Comment (bgodlin) [reply Ethereum/ethereum-bayc/schema.graphql:5]: https://github.com/subquery/ethereum-subql-starter/pull/42/commits/787557af34849d9c5b0b001f6c3ff4fbb68c1319

File: Ethereum/ethereum-bayc/schema.graphql (hunk 1 above)
Comment (bgodlin) [reply Ethereum/ethereum-bayc/schema.graphql:9]: https://github.com/subquery/ethereum-subql-starter/pull/42/commits/787557af34849d9c5b0b001f6c3ff4fbb68c1319

**Review by deverka — APPROVED** [review]
looks good to me

**Pull request merged**